.github/workflows/code-scan-cron.yml

```diff
@@ -0,0 +1,9 @@
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
You're always using the same schedule, would we get any benefit if we use a different one per project? Perhaps a few minutes appart?

Reply:
What would the benefit be? It's not running against our servers. Also, I changed every repo that isn't cli, certificates, crypto to only do once a week at midnight on sunday.